resources/ahk/TradeMacro.ahk

```diff
@@ -87,13 +87,6 @@ OpenSearchOnPoeTrade:
 	SuspendPOEItemScript = 1 ; This allows us to handle the clipboard change event
 	Send ^{sc02E}
 	Sleep 250
-	
-	TradeFunc_DoParseClipboard()
-	If (!Item.Name and TradeOpts.OpenUrlsOnEmptyItem) {
```

Comment:
It seems I was confused about that part you deleted, mistook it for a different part...

I'm not sure this wiki problem is no related to your changes, at least to some of the more recent ones. I know it worked.

The not initialized UrlAffix shouldn't be a problem, but simply initializing it doesn't hurt.